src/Maths/math.vector.ts

```diff
@@ -5134,6 +5156,97 @@ export class Matrix {
         );
     }
 
+    /**
+     * Gets a new rotation matrix used to rotate an entity so as it looks in the direction specified by forward from the eye position, the up direction being oriented like "up".
+     * This function works in left handed mode
+     * @param forward defines the forward direction
+     * @param up defines the up vector for the entity
+     * @returns the new matrix
+     */
+    public static LookDirectionLH(forward: DeepImmutable<Vector3>, up: DeepImmutable<Vector3>): Matrix {
+        var result = new Matrix();
+        Matrix.LookDirectionLHToRef(forward, up, result);
+        return result;
+    }
+
+    /**
+     * Sets the given "result" Matrix to a rotation matrix used to rotate an entity so that it looks in the direction of forward, the up direction being oriented like "up".
+     * This function works in left handed mode
+     * @param forward defines the forward direction
+     * @param up defines the up vector for the entity
+     * @param result defines the target matrix
+     */
+    public static LookDirectionLHToRef(forward: DeepImmutable<Vector3>, up: DeepImmutable<Vector3>, result: Matrix): void {
+        // First ensure that two vectors are valid and approximately orthogonal.
+        if (forward.length() < Epsilon || up.length() < Epsilon || Math.abs(Vector3.Dot(forward, up)) >= Epsilon) {

```

Comment:
I don't think we should have these checks, as as a general rule Babylon does not validate the input parameters passed to a function (for performance reasons): it's up to the caller to make sure that the parameters are valid.

Simply updating the doc to indicate that both vectors must be orthogonal (and probably of unit length? that would save normalizing in the function itself as most of the time the user will already have unit vectors) should be enough.

Reply:
Done!